### PR TITLE
Upgrade black to version ~=22.0

### PR DESCRIPTION
**Pull request by LysandreJik**
Upgrades the `huggingface_hub` library quality tool `black` to use the first stable release of `black`, version 22.0.

### CODE REVIEW

**Comment by muellerzr** [comment]
We should also update the [setup.py](https://github.com/huggingface/huggingface_hub/blob/main/setup.py#L41) so it trickles into the CI/CD

File: setup.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def get_version() -> str:
 ]
 
 extras["quality"] = [
-    "black==21.4b0",
+    "black~=22.0",
```
Comment (LysandreJik) [inline setup.py:41]: It is updated no?

Comment (muellerzr) [reply setup.py:41]: Yes it is, sorry I glanced over that! 

Peeking at it more, *eventually* this CI/CD may fail due to us not pinning the versions of black like we do with the setup.py https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/python-api-quality.yaml#L19

**Review by muellerzr — APPROVED** [review]

**Review by osanseviero — APPROVED** [review]
Thanks for the PR! 🔥 

cc people working in the repo @merveenoyan @nateraw @Narsil so you update your black version as well

**Pull request merged**